### PR TITLE
Fix tab switch for adding mastery checks

### LINKED ISSUE #243: [PE-D] Add Mastery Check: Does not redirect to Mastery Check List
I am unsure if this is a correct behaviour, but after adding a 'Mastery Check', there is a success message for it, but I'm still in the Consultation tab?

The Mastery Check is created if I check the tab manually, but this may be confusing for others since all the previous add commands before 'Mastery Check' redirected the user to the corresponding tab for the item added.

-------------
Labels: `severity.Low` `type.FunctionalityBug`
original: rachel170/ped#6

### PR DESCRIPTION
**Pull request by ngzhenteng**
Fix tab switch for adding mastery checks
Add TodoBuilder
Fixes #243

### CODE REVIEW

**Comment by wireseo** [comment]
Thank you so much for this! LGTM :)

File: src/main/java/seedu/jarvis/logic/parser/AddCommandParser.java [hunk 1]
```diff
@@ -93,25 +86,26 @@ public AddCommand parse(String args) throws ParseException {
...
-            }
-
-            Name name = ParserUtil.parseName(argMultimap.getValue(PREFIX_NAME).get());
-            Telegram telegram = ParserUtil.parseTelegram(argMultimap.getValue(PREFIX_TELEGRAM).get());
-            Email email = ParserUtil.parseEmail(argMultimap.getValue(PREFIX_EMAIL).get());
-
-            Student student = new Student(name, telegram, email);
-
-            return new AddCommand(student);
+            throw new ParseException(String.format(MESSAGE_INVALID_COMMAND_FORMAT, AddCommand.MESSAGE_ADD_USAGE));
```
Comment (wireseo) [inline src/main/java/seedu/jarvis/logic/parser/AddCommandParser.java:90]: Thank you for cleaning up dead code!

File: docs/AboutUs.md [hunk 2]
```diff
@@ -51,7 +51,7 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
 [[portfolio](team/ngzhenteng.md)]
 
 * Role: Developer
-* Responsibilities: Student Information Retrieval Features
```
Comment (peironggg) [inline docs/AboutUs.md:54]: Thanks for adding this feature!

File: src/main/java/seedu/jarvis/ui/MainWindow.java [hunk 3]
```diff
@@ -378,6 +378,8 @@ private CommandResult executeCommand(String commandText) throws CommandException
                 tabSelector.select(questTab);
             } else if (commandTargetFeature == CommandTargetFeature.Consultations) {
                 tabSelector.select(consultationTab);
+            } else if (commandTargetFeature == CommandTargetFeature.MasteryCheck) {
```
Comment (wireseo) [inline src/main/java/seedu/jarvis/ui/MainWindow.java:381]: Nice, thank you! This was what caused the error, huh.

File: src/test/java/seedu/jarvis/testutil/TodoBuilder.java [hunk 4]
```diff
@@ -0,0 +1,29 @@
+package seedu.jarvis.testutil;
+
```
Comment (wireseo) [inline src/test/java/seedu/jarvis/testutil/TodoBuilder.java:2]: Nice Builder for testing purposes!

**Pull request merged**